config.tf

```
@@ -1,6 +1,12 @@
+locals {
+  demo_or_es   = "${var.postgresql_address != "" ? "external_services" : "demo"}"
+  ag_or_ol     = "${var.airgap_package_url != "" ? "airgap" : "online" }"
```

if we change this to something like 
```suggestion
  app_network_type     = "${var.airgap_package_url != "" ? "airgap" : "online" }"
```

or something where we say what we're using it for, the variable would be easier to understand.